identity/app/actions/AuthenticatedActions.scala

```diff
@@ -73,9 +93,14 @@ class AuthenticatedActions(
     }
   }
 
+
+  def authAction: AuthenticatedBuilder[AuthenticatedUser] =
```

Comment:
I know we have left this in for compatibility, but for each place that uses this - we probably need to ask if we need to change the redirect behaviour there too.